core/src/main/java/com/amplitude/core/utilities/Diagnostics.kt

```diff
@@ -0,0 +1,47 @@
+package com.amplitude.core.utilities
+
+import java.util.Collections
+
+class Diagnostics() {
```

Comment:
Can we make this `internal` for now?

Reply:
That's what I started first, however, after change it to use in the constructor of EventsFileManager and refer that in android module I need to make this public